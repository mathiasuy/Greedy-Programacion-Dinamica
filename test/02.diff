1,2c1,14
< 1>Volumen: 216
< 2>Fin.
---
> ==4378== error calling PR_SET_PTRACER, vgdb might block
> ==4378== Conditional jump or move depends on uninitialised value(s)
> ==4378==    at 0x1090FE: hay_solapamiento(bool*, intervalo_t*, unsigned int) (in /mnt/c/p3/tareap3/principal)
> ==4378==    by 0x109579: main (in /mnt/c/p3/tareap3/principal)
> ==4378== 
> ==4378== Invalid read of size 1
> ==4378==    at 0x1090F9: hay_solapamiento(bool*, intervalo_t*, unsigned int) (in /mnt/c/p3/tareap3/principal)
> ==4378==    by 0x109579: main (in /mnt/c/p3/tareap3/principal)
> ==4378==  Address 0x5b7e321 is 0 bytes after a block of size 1 alloc'd
> ==4378==    at 0x4C3017F: operator new(unsigned long) (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
> ==4378==    by 0x109CA9: max_volumen(intervalo_t const*, unsigned int) (in /mnt/c/p3/tareap3/principal)
> ==4378==    by 0x109553: main (in /mnt/c/p3/tareap3/principal)
> ==4378== 
> 1>
\ No newline at end of file
