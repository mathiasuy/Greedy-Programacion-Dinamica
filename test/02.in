max_volumen_intervalos
17
3-10-30
1-15-10
15-16-13
20-30-1
30-35-1
35-45-1
46-51-10
50-52-5
49-52-30
50-53-50
52-55-10
55-58-20
57-62-50
59-63-50
61-64-80
62-65-70
63-70-40
Fin

