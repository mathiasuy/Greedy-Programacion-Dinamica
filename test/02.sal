==4601== Conditional jump or move depends on uninitialised value(s)
==4601==    at 0x1091FE: hay_solapamiento(bool*, intervalo_t*, unsigned int) (in /mnt/c/p3/tareap3ant/principal)
==4601==    by 0x109679: main (in /mnt/c/p3/tareap3ant/principal)
==4601== 
==4601== Invalid read of size 1
==4601==    at 0x1091F9: hay_solapamiento(bool*, intervalo_t*, unsigned int) (in /mnt/c/p3/tareap3ant/principal)
==4601==    by 0x109679: main (in /mnt/c/p3/tareap3ant/principal)
==4601==  Address 0x5b7e321 is 0 bytes after a block of size 1 alloc'd
==4601==    at 0x4C3017F: operator new(unsigned long) (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==4601==    by 0x109ED4: max_volumen(intervalo_t const*, unsigned int) (in /mnt/c/p3/tareap3ant/principal)
==4601==    by 0x109653: main (in /mnt/c/p3/tareap3ant/principal)
==4601== 
1>