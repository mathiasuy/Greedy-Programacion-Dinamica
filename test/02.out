1>Volumen: 216
2>Fin.
